### PR TITLE
chore: [#188631414] update tag styling

### PR DESCRIPTION
**Pull request by FarazA22**
### Approach
https://www.pivotaltracker.com/story/show/188631414

The intention behind this ticket is to update the styling for all default tabs in the application. The new styling will align with what is noted in the design system with one caveat, the blue line will continue to exist at the bottom instead of the top.

## Code author checklist

- [x] I have rebased this branch from the latest main branch
- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation on the engineering documentation website
- [x] I have not used any relative imports
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] I have checked for and removed instances of unused config from CMS
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see CMS Additions in Engineering Reference/FAQ on the engineering documentation site)
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

**Comment by FarazA22** [comment]
https://github.com/user-attachments/assets/8e3cb9ca-332b-429e-8cb7-fe1909f29dfd

File: web/src/lib/muiTheme.ts [hunk 1]
```diff
@@ -72,6 +72,32 @@ export default createTheme({
         },
       },
     },
+    MuiButtonBase: {
+      styleOverrides: {
+        root: {
+          "&.MuiTab-root": {
+            "&:focus": {
+              outline: ".25rem solid #38536f",
```
Comment (FarazA22) [inline web/src/lib/muiTheme.ts:80]: overrides green outline from uswds

File: web/src/lib/muiTheme.ts [hunk 2]
```diff
@@ -72,6 +72,32 @@ export default createTheme({
...
+            },
+          },
+        },
+      },
+    },
+    MuiTab: {
+      styleOverrides: {
+        root: {
+          color: "#38536f !important",
```
Comment (FarazA22) [inline web/src/lib/muiTheme.ts:89]: overrides text color

File: web/src/lib/muiTheme.ts [hunk 3]
```diff
@@ -72,6 +72,32 @@ export default createTheme({
...
+        },
+      },
+    },
+    MuiTabs: {
+      styleOverrides: {
+        indicator: {
+          height: "4px !important",
+          backgroundColor: "#38536f !important",
```
Comment (FarazA22) [inline web/src/lib/muiTheme.ts:97]: overrides indicator color and fixes the height

**Review by SirSamuelJoseph — APPROVED** [review]
Looks slick!

**Review by somabadri — APPROVED** [review]
Great work Faraz!

**Pull request merged**